tests/unit/OptionsListUtilsTest.ts

```diff
@@ -619,7 +627,7 @@ describe('OptionsListUtils', () => {
 
     it('getShareDestinationsOptions()', () => {
         // Filter current REPORTS as we do in the component, before getting share destination options
-        const filteredReports = Object.entries(REPORTS).reduce<Record<string, Report>>((reports, [reportKey, report]) => {
+        const filteredReports = Object.entries(REPORTS).reduce<Record<string, Report | null>>((reports, [reportKey, report]) => {
```

Comment:
```suggestion
        const filteredReports = Object.entries(REPORTS).reduce<Record<string, OnyxEntry<Report>>>((reports, [reportKey, report]) => {
```

or, if possible, could be

```ts
const filteredReports = Object.entries(REPORTS).reduce<OnyxCollection<Report>>((reports, [reportKey, report]) => {
```